Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,13 @@
+class GitWhenMerged < Formula
+  desc "Find the where a commit was merged in git."
+  homepage "https://github.com/mhagger/git-when-merged"
+  url "https://github.com/mhagger/git-when-merged/archive/v1.0.0.tar.gz"
+  sha256 "1be0f2c660e6e0cef66593aac681163b97027a7be028b6d2b3901fee00639ad7"
+  def install
```

Comment:
Please add `bottle :unneeded` one empty line under the `sha256` as well, after `def install` has been nudged downwards a bit. Cheers!